types/node-red/index.d.ts

```diff
@@ -1,221 +1,234 @@
-// Type definitions for node-red 0.20
-// Project: http://nodered.org
+// Type definitions for node-red 1.1
+// Project: https://github.com/node-red/node-red/tree/master/packages/node_modules/node-red, https://nodered.org/
 // Definitions by: Anders E. Andersen <https://github.com/andersea>
 //                 Thomas B. Mørch <https://github.com/tbowmo>
 //                 Bernardo Belchior <https://github.com/bernardobelchior>
+//                 Alex Kaul <https://github.com/alexk111>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 3.4
+// Minimum TypeScript Version: 3.1
 
-/// <reference types="node" />
+// tslint:disable:no-empty-interface no-mergeable-namespace
```

Comment:
```suggestion
// tslint:disable:no-empty-interface
```
can we remove this exclusion and remove the namespaces with same name? I understand `namespace X {}, namespace X.A {}` etc, but what's the purpose of those three blocks? IMO it just make things less readable (the outcome is same)

Reply:
Fixed. That was done to visually separate three different groups of interfaces and keep them in a single namespace at the same time. Emphasized with comments instead of the namespace duplicates.